src/java/org/apache/cassandra/index/sai/virtual/StorageAttachedIndexTables.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.index.sai.virtual;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.apache.cassandra.db.virtual.VirtualTable;
+
+public class StorageAttachedIndexTables
+{
+    private StorageAttachedIndexTables()
+    {}
+
+    public static Collection<VirtualTable> getAll(String keyspace)
+    {
+        return Arrays.asList(new IndexesSystemView(keyspace));
```

Comment:
```suggestion
        return Collections.singletonList(new IndexesSystemView(keyspace));
```
...not that this is performance intensive...but why not